data/transaction/status.go

```diff
@@ -70,3 +80,33 @@ func (params *StatusComputer) isDestinationMe() bool {
 func (params *StatusComputer) isContractDeploy() bool {
 	return core.IsEmptyAddress(params.Receiver) && len(params.TransactionData) > 0
 }
+
+// SetStatusIfIsRewardReverted will compute and set status for a reverted reward transaction
+func (params *StatusComputer) SetStatusIfIsRewardReverted(tx *ApiTransactionResult) bool {
```

Comment:
why do we have `params` here? (sc would have been more appropriate)

Reply:
done.